08-components-lab/src/01-timeconverter.jsx

```diff
@@ -8,5 +8,49 @@
 
 import React from 'react';
 
-export default React.createClass({
-});
+export default class TimeConverter extends React.Component{
+    constructor(props){
+        super(props);
+        this.state={
+            number : 0
+        }
+    }
+
+    static propTypes = {
+        number: React.propTypes.number.isRequired
+    };
+
+    getSeconds(){
+        return Number(this.state.number);
+    };
+    getMinutes(){
+        return Number(this.state.number / 60);
+    };
+    getHours(){
+        return Number(this.state.number / 3600);
+    }
+
+    calc = (evt) => {
+        //reset numbers before setting new val
+        this.setState({number:0});
+        if (evt.target.name === "seconds"){
```

Comment:
Instead of using 3 times ev.target.name and ev.target.value you can assign it to a const in the beginning of a method. Alternative is to use ES6 destructuring in arrow function arguments like this:
```
({ target: { value, name }) => {

}
```